README.md

```diff
@@ -14,34 +15,35 @@ CSS](https://github.com/zendeskgarden/css-components) for styling.
 See the individual package README for the React component you would like
 to install.
 
-Package | Version | Dependencies
```

Comment:
I think we could consider removing `autocomplete`, `menus`, and `selects` from this table as a move toward encouraging the behavior (use dropdowns, instead) we desire. Although, I guess `multiselect` isn't represented, yet.

Reply:
I've removed menus and select from the table. Lets leave `autocomplete` until the multiselect is moved over.

Reply:
This reminds me. We will need to add in the deprecation warnings to the docs and packages, but that can come after the PR.